.gitignore

```diff
@@ -1,6 +1,5 @@
 test/processed
 node_modules
-lib/
```

Comment:
Can you add this back so you don't commit the built files

Reply:
Done.